manga_ocr_dev/training/get_model.py

```diff
@@ -2,15 +2,14 @@
     AutoConfig,
     AutoModelForCausalLM,
     AutoModel,
-    TrOCRProcessor,
     VisionEncoderDecoderModel,
-    AutoFeatureExtractor,
+    AutoImageProcessor,
     AutoTokenizer,
     VisionEncoderDecoderConfig,
 )
 
 
-class TrOCRProcessorCustom(TrOCRProcessor):
+class TrOCRProcessorCustom:
```

Comment:
no need to derive from TrOCRProcessor, otherwise it will complain `feature_extractor, can't set attribute', as reported in #56